src/app.ts

```diff
@@ -52,12 +52,23 @@ async function checkGitInstalled() {
 }
 
 async function checkDirectoryIsGitRepository() {
-  const result = await execAsync(
-    "git rev-parse --is-inside-work-tree 2>/dev/null",
-    {
-      encoding: "utf-8",
-    }
-  );
+  const result = false;
+  if(process.platform === "win32") {
```

Comment:
We might not even need to check platforms here, running this command returns a string "true" if parsing, so looks like we don't need platform specific code anymore

Reply:
Are you saying, take out the check and just have the original code without the ```2>/dev/null``` code?

Reply:
pretty much, but of course with the check if it's the required output, to which I guess the 2>dev/null was there to catch any weird responses and make them a boolean false